.github/workflows/release-cli.yml

```diff
@@ -0,0 +1,98 @@
+name: Release new CLI versions
+on:
+  push:
+    branches: 
+      - "master"
+    paths:
+      - "cli/Makefile"
+env:
+  CLI_BUCKET: gs://cft-cli
```

Comment:
Are we using this in the Makefile? I also saw the same value being defined in the `cli/Makefile` as `BUCKET=gs://cft-cli`

Reply:
Yeah, we have it set in both places. We could remove from Makefile eventually but for now I'm leaving as-is.